### ISSUE TITLE
WQ+Coffea: Performance Regression?

### ISSUE THREAD

**Issue by dthain** [cooperative-computing-lab/cctools#2640]
@kmohrman reports that she was getting execution times of about 10 minutes with TopEFT and a chunksize of 100K on coffea-0.7.4, then upgraded coffea to 0.7.5 and performance went back to about 40 minutes.  Please investigate....

**Comment by kmohrman** [comment]
Thank you, @dthain! @btovar, here is a bit more information about the situation in case it helps to identify the issue:
* I've pasted the contents of the config file I was using for this test [here](https://gist.github.com/kmohrman/66cab62509c05ad6fbf3f322d71a15e4); the samples listed in that file are our privately produced EFT samples, but I saw the same issue when processing samples produced centrally by CMS. 
* I'm running on the master branch of [topcoffea](https://github.com/TopEFT/topcoffea). 
* I was not modifying any of the `executor_args` from the numbers that are committed on the master branch, and I did not change the chunk size from the default.
* The issue does not seem to affect the futures executor (I was getting fairly similar run times with 0.7.4 and 0.7.5, though when running with 0.7.5 it was sometimes a few minutes longer, the largest difference was only about 15%, so it was a much smaller difference than the factors of about 3 to 4 that I saw when running with wq).
* From the command line I was running:
```
time python work_queue_run.py path/to/my.cfg
```
Where the contents of `my.cfg` are pasted into the gist I linked above. The `condor_submit_workers` command I was running is the following:
```
condor_submit_workers -M ${USER}-workqueue-coffea -t 900 --cores 12 --memory 48000 --disk 150000 20
```
Please let me know if I can provide any other information! Thank you!

**Comment by kmohrman** [comment]
Now that coffea 0.7.6 has been released, I tried processing the set of samples linked above using 0.7.6, and the issue with the slow runtime seems to be resolved! The runtime matched what we had in 0.7.4.

**Comment by btovar** [comment]
Thanks for letting us know!  
Given the time it took in 0.7.5, my very strong hunch is that you were experiencing the wq executor incorrectly handling the compression parameter, as sometimes we checked for 'None', and 'False', and disabled it with 0, which was a mistake. 0.7.6 only disables compression with None, and accepts any other numeric value. 

False is treated like 0, and 0 means the minimum compression, but that's something common to all executors, and probably should be checked by run_uproot_job, as it is a quirck of lz4f.
